src/main/java/software/amazon/event/kafkaconnector/logging/ContextAwareLoggerFactory.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.event.kafkaconnector.logging;
+
+import java.io.IOException;
+import java.util.Properties;
+import org.apache.kafka.common.utils.LogContext;
+import org.slf4j.Logger;
+
+public class ContextAwareLoggerFactory {
```

Comment:
nit: can you please add a doc string here since it's not clear what this thing does right now/why we use it. You can keep it generic e.g., something like "adds context relevant information to the log output, such as the Git short commit SHA this connector was built from"